lib/api/address/v1/service.ts

```diff
@@ -27,6 +27,21 @@ export const searchAddress = (postCode: string): Promise<SearchAddressRespone> =
       return res;
     });
 
+export const getAddressViaUprn = (UPRN: string): Promise<SearchAddressResponse> =>
+  axiosInstance
+    .get<AddressAPIResponse>(`${config.addressApiUrlV1}/addresses?uprn=${UPRN}`, {
+      headers: {
+        "skip-x-correlation-id": true,
```

Comment:
All the other calls to APIs seem to have this key set up in the headers, so I've left it there (not entirely sure why it's required)

Reply:
That's fine but could you add some tests around this please?